lib/OnyxUtils.ts

```diff
@@ -295,6 +298,32 @@ function isSafeEvictionKey(testKey: OnyxKey): boolean {
     return evictionAllowList.some((key) => isKeyMatch(key, testKey));
 }
 
+/**
+ * Stores a connection ID associated with a given key.
+ *
+ * @param connectionID - a connection ID of the subscriber
+ * @param key - a key that the subscriber is connected to
+ */
+function storeKeyByConnections(key: OnyxKey, connectionID: number) {
+    if (!onyxKeyToConnectionIDs.has(key)) {
+        onyxKeyToConnectionIDs.set(key, []);
+    }
+    onyxKeyToConnectionIDs.get(key).push(connectionID);
+}
+
+/**
+ * Deletes a connection ID associated with its corresponding key.
+ *
+ * @param {number} connectionID - The connection ID to be deleted.
+ */
+function deleteKeyByConnections(connectionID: number) {
+    const subscriber = callbackToStateMapping[connectionID];
+
+    if (subscriber && onyxKeyToConnectionIDs.has(subscriber.key)) {
+        onyxKeyToConnectionIDs.set(subscriber.key, _.without(onyxKeyToConnectionIDs.get(subscriber.key), connectionID));
```

Comment:
Thanks for point this out, I also improve it using `filter` instead of `without`

<img width="974" alt="Screenshot 2024-05-10 at 13 16 37" src="https://github.com/Expensify/react-native-onyx/assets/1676818/bd864e89-51a7-4b3a-a174-e7770082a530">

Reply:
Sorry, don't want to be the nit picker here - I had a typo in my first example, it should be `updatedConnectionIDs` with a d instead of an s 😅 @gedu

Reply:
Don't worry, this is the moment to change it 💪